src/encoded/schemas/manatee_donor.json

```diff
@@ -0,0 +1,170 @@
+{
+    "title": "Manatee donor",
+    "description": "Schema for submitting a manatee individual.",
+    "comment": "The manatee donor object is meant to represent an individual manatee organism.",
```

Comment:
@iyoungworth I am not sure manatee's donors represent an individual animal, or a "strain"?